README.md

```diff
@@ -15,6 +15,75 @@ If you can't find what you are looking for, try the [old docs on the ODK website
+   
+   More info at the [Docker CE docs page](https://docs.docker.com/install/)
+ * Install [Git-LFS](https://git-lfs.github.com/).
+
+### Cloning the repo
+
+After that, you need to clone the docs repo
+
+```
+git clone https://github.com/opendatakit/docs.git
+```
+
+It can take a long time (>10 minutes) to clone the repo due to the large number of images in the docs. If you get an error such as `Smudge error` or `github's rate limit reached`, run `git checkout -f HEAD` until you get the message `Checking out files: 100% done`.
+
+### Building the Docker image
+
+Next, you need to build the Docker image with all the tools you will be using to work with ODK's docs.
+
+```
+docker build -t odk-docs .
+```
+
+It can take a long time to build the Docker image, but you only need to do this once.
+
+**Windows users**
+ * All commands should be run in an elevated PowerShell window. Right click on it and select the "Run as administrator" option.
+ * Ensure Docker is running by checking your system tray. If Docker is not running, launch "Docker for Windows" app and wait until a notification confirms that Docker is running.
+
+### Building and serving the docs locally 
+
+Build and serve the docs locally with:
+ * Windows: `.\run-task.bat odk1-autobuild`
+ * Linux/macOS: `./run-task.sh odk1-autobuild`
+ 
+You can then view the docs in your browser at http://localhost:8000.
+
+Press `ctrl-C` on your keyboard to stop the build server. It could take a while to effectively stop, and you can always close the Powershell window.
+
+If you get a `The name "odk-docs" is already in use by container` error message, run the following command:
+
+```
+docker kill odk-docs
+```
+
+### Other build tasks  
+
+You can also use the `run-task` script to build both ODK and ODK2 docs, or to run just a portion of the build process.
+
+For both ODK 1 and ODK 2:
+
+|          |    Build     |     Clean     |     Check Style & Spell     |    Test    |
+| -------- | :---------:  | :-----------: | :-------------------------: | :--------: |
+| **Options** | build-all |  clean |  check-all | test |
+
+For a specific ODK version:
+
+|          | Build & Serve | Build |    Copy     |     LaTeX     |     Style Check     |    Spell Check     |    Check All   |
+| -------- | :-----------: | :---: | :---------: | :-----------: | :-----------------: | :----------------: | :----------------: |
+| **Options** | odk1-autobuild | odk1-build | odk1-copy |  odk1-latex |  odk1-style-check | odk1-spell-check |    odk1-check     |
+
+To build ODK 2 docs, just replace `odk1` with `odk2`. 
```

Comment:
s/just//g. Just, simply, etc, is language we want to avoid because it implies something might be easy. It might not be.